packages/cacti-copm-test/src/main/typescript/corda/copm-tester-corda.ts

```diff
@@ -0,0 +1,230 @@
+import { LogLevelDesc, Logger } from "@hyperledger/cactus-common";
+import * as fs from "fs";
+import * as path from "path";
+import { createPromiseClient, PromiseClient } from "@connectrpc/connect";
+import { createConnectTransport } from "@connectrpc/connect-node";
+
+import { CopmTester } from "../interfaces/copm-tester";
+import { TestAssets } from "../interfaces/test-assets";
+import { CopmNetworkMode } from "../lib/types";
+import { TestAssetsCordaCli } from "./test-assets-corda-cli";
+import { PluginRegistry } from "@hyperledger/cactus-core";
+import { ConfigService } from "@hyperledger/cactus-cmd-api-server";
+import { AuthorizationProtocol } from "@hyperledger/cactus-cmd-api-server";
+import { v4 as uuidV4 } from "uuid";
+import { PluginCopmCorda } from "@hyperledger-cacti/cacti-plugin-copm-corda";
+import { AddressInfo } from "node:net";
+import { Servers } from "@hyperledger/cactus-common";
+import http from "node:http";
+import { ApiServer } from "@hyperledger/cactus-cmd-api-server";
+import { CopmCordaContainer } from "./copm-corda-container";
+import { CordaRPCConfig } from "./corda-rpc-config";
+
+export class CopmTesterCorda implements CopmTester {
+  logLevel: LogLevelDesc = "INFO";
+  log: Logger;
+  contractNames: CopmContractNames;
+  private weaverRelativePath = "../../../../../../weaver/";
+  private packageRelativePath = "../../../../";
+  private networkMode: CopmNetworkMode;
+  private testAssetMap = new Map<string, TestAssetsCordaCli>();
+  private httpServer: http.Server | null;
+  private apiServer: ApiServer | null;
+  private serverAddress: string;
+  private copmCordaContainer: CopmCordaContainer;
+
+  constructor(log: Logger, networkMode: CopmNetworkMode) {
+    this.log = log;
+    this.networkMode = networkMode;
+    this.apiServer = null;
+    this.httpServer = null;
+    this.serverAddress = "";
+    this.contractNames = {
+      pledgeContract: "org.hyperledger.cacti.weaver.imodule.corda.flows",
+      lockContract: "org.hyperledger.cacti.weaver.imodule.corda.flows",
+    };
+    this.copmCordaContainer = new CopmCordaContainer({
+      logLevel: this.logLevel,
+      rpcConfigJson: this.getRPCConfig(),
+      relayConfigJson: this.getRelayConfig(),
+      remoteNetworkConfigJson: this.getRemoteNetworkConfig(),
+    });
+  }
+
+  public async assetsFor(account: DLAccount): Promise<TestAssets> {
+    const accountKey = account.userId + "@" + account.organization;
+    if (!this.testAssetMap.has(accountKey)) {
+      this.testAssetMap.set(accountKey, await this.createTestAssets(account));
+    }
+    const assets = this.testAssetMap.get(accountKey);
+    if (!assets) {
+      throw Error(`no assets found for account ${accountKey}`);
+    }
+    return assets;
+  }
+
+  private async createTestAssets(
+    account: DLAccount,
+  ): Promise<TestAssetsCordaCli> {
+    return new TestAssetsCordaCli(account, this.log);
+  }
+
+  public networkNames(): string[] {
+    return ["Corda_Network", "Corda_Network2"];
```

Comment:
I wonder if this should be hardcoded or lifted from a configuration file. @sandeepnRES ?

Reply:
I see the user identities and org names are also hardcoded below. I looked at the older code, where such values were hardcoded in the `test` folders of the `corda-interop-app` module, so if this package is the equivalent of that test suite, then I'm fine with the hardcoding.